code/modules/mob/living/say.dm

```diff
@@ -231,8 +228,13 @@ proc/get_radio_key_from_channel(var/channel)
 	var/list/handle_v = handle_speech_sound()
 	var/sound/speech_sound = handle_v[1]
 	var/sound_vol = handle_v[2]
+	var/italics
+
+	if(src.is_diona())

```

Comment:
This is going to be a really bad idea.

You have
```dm
/mob/living/carbon/human/handle_speech_problems()
```

as a function. Just make that call a species function _in all cases_. Distinguishing between diona and the rest should not be done with a runtime check here, but instead, via inheritance.